core/src/main/java/io/questdb/cairo/IndexBuilder.java

```diff
@@ -63,7 +63,7 @@ public void close() {
     }
 
     @Override
-    protected boolean checkColumnType(TableReaderMetadata metadata, int rebuildColumnIndex) {
+    protected boolean checkColumnType(BaseRecordMetadata metadata, int rebuildColumnIndex) {
```

Comment:
what does this function mean? one impl checks if column is indexed, the other checks if type is String